public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -207,83 +205,87 @@ include ../../../../_includes/_util-fns
 
 +makeExample('dependency-injection/ts/app/heroes/hero.service.1.ts',null, 'app/heroes/hero.service.ts' )
 :marked
-  Our `HeroService` exposes a `getHeroes()` method that returns
-  the same mock data as before but none of its consumers need to know that.
+  Our `HeroService` exposes a `getHeroes` method that returns
+  the same mock data as before, but none of its consumers need to know that.
 .l-sub-section
   :marked
     We aren't even pretending this is a real service.
     If we were actually getting data from a remote server, the API would have to be asynchronous,
     perhaps returning
-    [ES2015 promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+    [ES2015 promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
     We'd also have to rewrite the way components consume our service.
-    This is important but not important to our current story.
+    This is important in general, but not to our current story.
 :marked
   A service is nothing more than a class in Angular 2.
   It remains nothing more than a class until we register it with an Angular injector.
 
-  ### Configuring the Injector
+  ### Configuring the injector
 
   <a id="bootstrap"></a>
   We don't have to create an Angular injector.
   Angular creates an application-wide injector for us during the bootstrap process.
 
-+makeExample('dependency-injection/ts/app/main.ts', 'bootstrap', 'app/main.ts (bootstrap)')(format='.')
++makeExample('dependency-injection/ts/app/main.ts', 'bootstrap', 'app/main.ts (excerpt)')(format='.')
 :marked
-  We do have to configure the injector by registering the **providers** 
-  that create the services we need in our application.
+  We do have to configure the injector by registering the **providers**
+  that create the services that our application needs.
   We'll explain what [providers](#providers) are later in this chapter.
   Before we do, let's see an example of provider registration during bootstrapping:
-  
+  <!-- Again we show an example of bad code. Ew. I see we quickly explain
+  what you should do instead, but people might just copy the code and miss the explanation. -->
```

Comment:
If they blindly copy it, presumably they copy the comment with it. One can only hope that a code reviewer says "What is _that_?"

Unlike other examples of "bad" code, this example is merely discourage. It still works. Believe it or not, this was once the _encouraged_ practice ;-)